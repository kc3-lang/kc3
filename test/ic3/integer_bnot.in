quote ~ -170141183460469231731687303715884105728
~ -170141183460469231731687303715884105728
quote ~ -170141183460469231731687303715884105727
~ -170141183460469231731687303715884105727
quote ~ -9223372036854775808
~ -9223372036854775808
quote ~ -9223372036854775807
~ -9223372036854775807
quote ~ -2147483648
~ -2147483648
quote ~ -2147483647
~ -2147483647
quote ~ -32768
~ -32768
quote ~ -32767
~ -32767
quote ~ -128
~ -128
quote ~ -127
~ -127
quote ~ 255
~ 255
quote ~ 256
~ 256
quote ~ 65535
~ 65535
quote ~ 65536
~ 65536
quote ~ 4294967295
~ 4294967295
quote ~ 4294967296
~ 4294967296
quote ~ 18446744073709551615
~ 18446744073709551615
quote ~ 18446744073709551616
~ 18446744073709551616
quote ~ 340282366920938463463374607431768211455
~ 340282366920938463463374607431768211455
quote ~ 340282366920938463463374607431768211456
~ 340282366920938463463374607431768211456
