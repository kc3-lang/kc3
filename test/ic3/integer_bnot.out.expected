~ -170141183460469231731687303715884105728
170141183460469231731687303715884105727
~ -170141183460469231731687303715884105727
170141183460469231731687303715884105726
~ -9223372036854775808
9223372036854775807
~ -9223372036854775807
9223372036854775806
~ -2147483648
2147483647
~ -2147483647
2147483646
~ -32768
32767
~ -32767
32766
~ -128
127
~ -127
126
~ 255
0
~ 256
65279
~ 65535
0
~ 65536
4294901759
~ 4294967295
0
~ 4294967296
18446744069414584319
~ 18446744073709551615
0
~ 18446744073709551616
340282366920938463444927863358058659839
~ 340282366920938463463374607431768211455
0
~ 340282366920938463463374607431768211456
115792089237316195423570985008687907852929702298719625575994209400481361428479
